test/unit/test-plugin/src/index.ts

```diff
@@ -1,22 +1,25 @@
+// @ts-ignore: this is a virtual module which we mock in the test so it doesn't exist when tsc runs
```

Comment:
When this line isn't here, tsc says "can't find module code-server" and then it doesn't compile and the tests fail. Any ideas on a better fix? @code-asher

Reply:
Nevermind, I figured it out